CRM/Contact/Form/Task.php

```diff
@@ -94,12 +94,14 @@ public function preProcess() {
    * Common pre-processing function.
    *
    * @param CRM_Core_Form $form
-   * @param bool $useTable
    */
-  public static function preProcessCommon(&$form, $useTable = FALSE) {
+  public static function preProcessCommon(&$form) {
     $form->_contactIds = array();
     $form->_contactTypes = array();
 
+    $formName = CRM_Utils_System::getClassName($form->controller->getStateMachine());
+    $useTable = ($formName !== 'CRM_Export_StateMachine_Standalone') ? TRUE : FALSE;
```

Comment:
@eileenmcnaughton seems `$useTable` was default to false for all function call, but is now only for `CRM_Export_StateMachine_Standalone`.

This doesn't allow profile to load on normal `update multiple` action from search results.

![image](https://user-images.githubusercontent.com/5929648/41701200-2eddd930-7549-11e8-9c01-5813eee17921.png)

Not sure if the operator be `==` OR the assignment of true false be reversed? Thoughts?

Reply:
Tested and raised https://github.com/civicrm/civicrm-core/pull/12349 which works fine and seems to follow the previous assignment of this variable.